Files/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -13,15 +13,55 @@
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
     mc:Ignorable="d">
     <StackPanel Spacing="12">
-        <TextBlock
-            x:Uid="DrivesWidgetDescription"
-            HorizontalAlignment="Left"
-            FontFamily="Segoe UI"
-            FontSize="14"
-            FontWeight="Medium"
-            Opacity=".8"
-            Text="Drives" />
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition />
+                <ColumnDefinition />
+            </Grid.ColumnDefinitions>
+            
+            <TextBlock x:Uid="DrivesWidgetDescription"
+                       HorizontalAlignment="Left"
+                       FontFamily="Segoe UI"
+                       FontSize="14"
+                       FontWeight="Medium"
+                       Opacity=".8"
+                       Text="Drives" />
 
+            <!--  Map Drive button  -->
+            <Button x:Uid="DrivesWidgetMoreOptionsButton"
+                    Grid.Column="1"
+                    Width="29"
+                    Height="25"
+                    Padding="0"
+                    HorizontalAlignment="Right"
+                    VerticalAlignment="Center"
+                    HorizontalContentAlignment="Center"
+                    VerticalContentAlignment="Center"
+                    Background="Transparent"
+                    Style="{StaticResource ButtonRevealStyle}"
+                    ToolTipService.ToolTip="More options...">
```

Comment:
Being that there is no text here, this button probably needs AutomationProperties.